app/helpers/namespaces_helper.rb

```diff
@@ -5,7 +5,7 @@ def can_manage_namespace?(namespace)
   end
 
   def can_create_namespace?
-    current_user.admin? || APP_CONFIG.enabled?("user_permission.create_namespace")
+    Registry.any? && (current_user.admin? || APP_CONFIG.enabled?("user_permission.create_namespace"))
```

Comment:
Can you add a test case for this change ?

Reply:
To be honest this is not needed. I guess in the past we weren't forcing the user to fill the registry form and they could visit other pages. I'll just remove it.